extensions/applicationinsights-clickanalytics-js/README.md

```diff
@@ -41,106 +48,269 @@ appInsights.loadAppInsights();
-| parentDataTag | string | null | Stop traversing up the DOM to capture content name and value of elements when encountered with this tag. |
-| dntDataTag | string | ai-dnt | Custom attribute Tag to not track telemetry data. |
+| Name                      | Type    | Default   | Description                                                                                                                                                                              |
+| ------------------------- | ------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| useDefaultContentName     | boolean | false     | When a particular element is not tagged with default customDataPrefix or customDataPrefix is not provided by user, this flag is used to collect standard HTML attribute for contentName. |
+| customDataPrefix          | string  | `data-`   | Automatic capture content name and value of elements which are tagged with provided prefix.                                                                                              |
+| aiBlobAttributeTag        | string  | `ai-blob` | Plugin supports a JSON blob content meta data tagging instead of individual `data-\*` attributes.                                                                                        |
+| metaDataPrefix            | string  | null      | Automatic capture HTML Head's meta element name and content with provided prefix.                                                                                                        |
+| captureAllMetaDataContent | string  | null      | Automatic capture all HTML Head's meta element names and content. Default is false. If enabled this will override provided metaDataPrefix.                                               |
+| parentDataTag             | string  | null      | Stop traversing up the DOM to capture content name and value of elements when encountered with this tag.                                                                                 |
+| dntDataTag                | string  | `ai-dnt`  | HTML elements with this attribute will be ignored by the plugin for capturing telemetry data.                                                                                            |
 
-## Usage with Example Config (JS)
+## behaviorValidator
 
-```html
-<script type="text/javascript" src="https://js.monitor.azure.com/scripts/b/ai.2.min.js"></script>
-<script type="text/javascript" src="cdnPathToCLickAnalyticsJS"></script>
+There are three different behaviorValidator callback functions exposed as part of this extension. Users can also bring their own callback functions if the exposed functions donot solve their requirements. The basic idea is to bring your own behaviors datastructure and plugin uses this validator function while extracting the behaviors from the data tags.
```

Comment:
donot -> do not

Reply:
change "functions donot solve their requirements" -> "functions **do not** solve **your** requirements"